File: Models/Account.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace webtruyenBackEndAPI.Models
{
	[Table("account")]
	public class Account
	{
		[Key]
		[Column("account_id")]
		public string AccountId { get; set; } = Guid.NewGuid().ToString();
		[Column("mail")]
		public string Mail { get; set; }
		[Column("password")]
		public string Password { get; set; }
		[Column("user_name")]
		public string? UserName { get; set; }
		[Column("image")]
		public string? Image { get; set; }
		[Column("position")]
		public bool Position { get; set; } = false;

		public virtual ICollection<Follow> Follows { get; set; }
	}

}

File: src/main/java/com/webtruyenapi/entity/Account.java
package com.webtruyenapi.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

@Entity
@Table(name = "account")
@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
public class Account {
    @Id
    @Column(name = "account_id", length = 36)
    private String accountId;

    @Column(name = "mail", nullable = false, unique = true)
    private String mail;

    @Column(name = "password", nullable = false, columnDefinition = "TEXT")
    private String password;

    @Column(name = "user_name")
    private String userName;

    @Column(name = "image")
    private String image;

    @Column(name = "position")
    private Boolean position = false;

    @Column(name = "role")
    @Enumerated(EnumType.STRING)
    private Role role;

    @Column(name = "created_at")
    private LocalDateTime createdAt;

    @Enumerated(EnumType.STRING)
    private AccountStatus status;

    @Column(name = "current_token")
    private String currentToken;

    @OneToMany(mappedBy = "account", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Follow> follows;

    @PrePersist
    protected void onCreate() {
        if (this.accountId == null) {
            this.accountId = UUID.randomUUID().toString();
        }
        if (this.position == null) {
            this.position = false;
        }
    }
}
